lib/api.js

```diff
@@ -83,8 +83,8 @@ class RPC {
   start(configPath, callback, unsafeFlag=false) {
     let config = null;
 
-    if (this.shares.size + 1 >= cpus().length && !unsafeFlag) {
-      return callback(new Error('insufficient system resources available'));
+    if (this.shares.size >= cpus().length && !unsafeFlag) {
```

Comment:
I don't think so; this is in the start function so, by the end, it's modifying the state from 0 shares to +1 shares. Comparison is 0 farmers >= 1 core, unless I'm chasing my state management tail somewhere here.